itou/employee_record/management/commands/_common.py

```diff
@@ -0,0 +1,75 @@
+from io import BytesIO
+from os import path
+from typing import Optional
+
+import pysftp
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from rest_framework.renderers import JSONRenderer
+
+
+# Global SFTP connection options
+
+connection_options = None
+
+if settings.ASP_FS_KNOWN_HOSTS and path.exists(settings.ASP_FS_KNOWN_HOSTS):
+    connection_options = pysftp.CnOpts(knownhosts=settings.ASP_FS_KNOWN_HOSTS)
+
+
+class EmployeeRecordTransferCommand(BaseCommand):
+    def get_sftp_connection(self) -> pysftp.Connection:
+        """
+        Get a new SFTP connection to remote server.
+        """
+        return pysftp.Connection(
+            host=settings.ASP_FS_SFTP_HOST,
+            port=int(settings.ASP_FS_SFTP_PORT),
+            username=settings.ASP_FS_SFTP_USER,
+            private_key=settings.ASP_FS_SFTP_PRIVATE_KEY_PATH,
+            cnopts=connection_options,
+        )
+
+    def upload_json_file(self, json_data, conn: pysftp.Connection, dry_run=False) -> Optional[str]:
```

Comment:
TIL function annotations https://stackoverflow.com/questions/14379753/what-does-mean-in-python-function-definitions